sdk/storage/azure-storage-file-datalake/pom.xml

```diff
@@ -36,6 +36,9 @@
     <src.dir>src/main/java</src.dir>
     <test.dir>src/test/java</test.dir>
     <AZURE_TEST_SYSTEM_PLAYBACK_PROPERTIES_FILE>../azure-storage-common/ci.system.properties</AZURE_TEST_SYSTEM_PLAYBACK_PROPERTIES_FILE>
+
+    <!-- Configures the Surefire JVM to use a 6gb max heap. -->
+    <surefireXmx>4096m</surefireXmx>
```

Comment:
6gb != 4096mb

I'd just drop these comments, the xml element seems to be clear enough.